tests/scr/testSysbox

```diff
@@ -16,6 +16,7 @@ function idmapped_mount_supported() {
 	local rel_major=$(echo ${kernel_rel} | cut -d'.' -f1)
 	local rel_minor=$(echo ${kernel_rel} | cut -d'.' -f2)
 	[ ${rel_major} -gt 5 ] || ([ ${rel_major} -eq 5 ] && [ ${rel_minor} -ge 12 ])
+	[ $(docker info | grep "Backing Filesystem" | cut -d " " -f5) != "xfs" ] && return 0
```

Comment:
Question: don't we need to link lines 18 & 19 into a single statement? What happens for example if line 18 returns false, would the function as a whole immediately return false (as it should) or would it continue into line 19 (which would be a mistake).

Reply:
I think you're right need to check this furthermore. But it might create some false positive scenarios. Since in line 19 we have a return statement we could swap 18 and 19 between them ? So if the filesystem is not xfs it will move on and check the kernel version. Maybe needs an improvement like 
```
[ $(docker info | grep "Backing Filesystem" | cut -d " " -f5) != "xfs" ] && return 0 || return 1
```
And then the lines could be 
```
	local rel_major=$(echo ${kernel_rel} | cut -d'.' -f1)
	local rel_minor=$(echo ${kernel_rel} | cut -d'.' -f2)
	[ $(docker info | grep "Backing Filesystem" | cut -d " " -f5) != "xfs" ] && return 0 || return 1
	[ ${rel_major} -gt 5 ] || ([ ${rel_major} -eq 5 ] && [ ${rel_minor} -ge 12 ])
```

I have not tested that but it could provide the exit factor that we need. So a testing scenario would be a system with the kernel lower than 5.12 and a non xfs filesystem, right ?

Reply:
Now that I am thinking it even more I don't think the aboce proposal would work. I would return solely the outcome of the filesystem type only.